.github/scripts/js/ci.js

```diff
@@ -79,116 +79,262 @@ module.exports.updateCommentOnStart = async ({ github, context, core, name }) =>
 };
 
 /**
- * Update a comment in "release" issue with status of job or workflow.
+ * Return a human-readable duration between started_at and now.
  *
+ * @param started_at - A Unix timestamp.
+ * @returns {string}
+ */
+const humanDurationFrom = (started_at) => {
+  let res = '';
+
+  const start = parseInt(started_at, 10);
+  let d = Math.floor(Date.now() / 1000) - start;
+
+  // Seconds
+  const s = d % 60;
+  res = `${s}s`;
+
+  if (d >= 60) {
+    // Minutes
+    const m = ((d - s) / 60) % 60;
+    res = `${m}m${res}`;
+
+    if (d >= 3600) {
+      // Hours
+      const h = Math.floor(d / 3600);
+      res = `${h}h${res}`;
+    }
+  }
+
+  return res;
+}
```

Comment:
We could install node dependencies to use convenient lib for this kind of tasks. E.g. date-fns

https://date-fns.org/v2.28.0/docs/formatDistanceStrict